### PR TITLE
Add page for token endpoint reference

### LINKED ISSUE #1726: Document the endpoints for the `token` module.
### Description
Document the endpoints for the `token` module.

#### Affected branch / URL
`development`

#### Page location
`lisk-docs/api/`

### Additional resources
https://github.com/LiskHQ/lisk-sdk/blob/development/framework/src/modules/token/endpoint.ts

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #1726

### CODE REVIEW

File: docs/modules/ROOT/pages/api/module-rpc-api/token-endpoints.adoc [hunk 1]
```diff
@@ -0,0 +1,570 @@
...
+  "supportedTokens": [
+    "0200000000000000",
+    "0200000000000000"
+  ]
+}
+----
+====
+
+== `token_isSupported`
+Checks, if a specific token is supported by the network.
```
Comment (chris529) [inline docs/modules/ROOT/pages/api/module-rpc-api/token-endpoints.adoc]:
```suggestion
Checks if a specific token is supported by the network.
```

File: docs/modules/ROOT/pages/api/module-rpc-api/token-endpoints.adoc [hunk 2]
```diff
@@ -0,0 +1,570 @@
...
+      "amount": "0",
+      "tokenID": "0200000000000000"
+    }
+  ]
+}
+----
+====
+
+== `token_getInitializationFees`
+Return the fees for initializing new user and escrow accounts.
```
Comment (chris529) [inline docs/modules/ROOT/pages/api/module-rpc-api/token-endpoints.adoc]:
```suggestion
Returns the fees for initializing new user and escrow accounts.
```

File: docs/modules/ROOT/pages/api/module-rpc-api/token-endpoints.adoc [hunk 3]
```diff
@@ -0,0 +1,570 @@
...
+----
+{
+  "userAccount": "5000000",
+  "escrowAccount": "5000000"
+}
+----
+====
+
+== `token_hasUserAccount`
+Checks, if an account exists for the provided address and token in the network.
```
Comment (chris529) [inline docs/modules/ROOT/pages/api/module-rpc-api/token-endpoints.adoc]:
```suggestion
Checks if an account exists for the provided address and token in the network.
```

File: docs/modules/ROOT/pages/api/module-rpc-api/token-endpoints.adoc [hunk 4]
```diff
@@ -0,0 +1,570 @@
...
+[source,json]
+----
+{
+  "exists": true
+}
+----
+====
+
+== `token_hasEscrowAccount`
+Checks, if an escrow account exists for the provided chain ID and token in the network.
```
Comment (chris529) [inline docs/modules/ROOT/pages/api/module-rpc-api/token-endpoints.adoc]:
```suggestion
Checks if an escrow account exists for the provided chain ID and token in the network.
```

**Review by chris529 — CHANGES_REQUESTED** [review]
Looks great! Just a few very minor points 👍

**Review by chris529 — APPROVED** [review]
LGTM! 👍

**Review by bobanm — APPROVED** [review]

**Pull request merged**